content/en/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 date: 2023-02-09T10:21:41-06:00
 intro: WELCOME TO ROTATIONAL
-header: YOUR GO-TO AI <br /> INNOVATION PARTNER
+header: YOUR GO-TO AI <span>INNOVATION PARTNER</span>
```

Comment:
Using a span instead of a `<br />` element is a better practice and more accessible.